examples/timestamping/backend/src/schema.rs

```diff
@@ -58,20 +61,24 @@ impl<T> Schema<T>
 where
     T: AsRef<dyn Snapshot>,
 {
+    /// Returns the `ProofMapIndex` of timestamps with their proofs.
```

Comment:
Mutable version of this method has no mention of `proofs`. I suggest either remove or keep it in both cases